packages/data-stores/src/reader/helpers/index.ts

```diff
@@ -82,4 +82,26 @@ const applyCallbackToPages = < K extends string, T >(
 	};
 };
 
-export { callApi, applyCallbackToPages, getSubkey };
+// Helper function to determine which API endpoint to call based on whether the user is logged in or not.
+const getApiParams = (
```

Comment:
Since the file (i.e. `reader/helpers/index.ts`) is not specific to site subscriptions only, I would make the function's name a bit more specific. Maybe `getSubscriptionMutationParams`? 🤔

Reply:
Thanks for catching that, Maciej! I missed the fact the helper file is "Reader-general". 😬 I will update the name in a new commit. 🙂

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/77252/commits/a5457d66f41bf36c6163ef55c0048110352c4c73.